spark/build.sbt

```diff
@@ -0,0 +1,9 @@
+test in assembly := {}
+
+assemblyMergeStrategy in assembly := {
+  case "reference.conf" => MergeStrategy.concat
+  case "application.conf" => MergeStrategy.concat
+  case "META-INF/MANIFEST.MF" => MergeStrategy.discard
+  case "META-INF\\MANIFEST.MF" => MergeStrategy.discard
+  case _ => MergeStrategy.first
+}
```

Comment:
Why put this here? I avoided using subproject `build.sbt` because it makes things hard to locate.